pkg/contract/allocations.go

```diff
@@ -235,3 +235,39 @@ func GetEVMSubnetGenesisNativeMinterAdmin(
 	}
 	return getGenesisNativeMinterAdmin(app, network, genesisData)
 }
+
+func ContractAddressIsInBlockchainGenesis(
```

Comment:
Could we add unit tests for these two new functions?

Reply:
one is not used, adding to the other